src/parser/jobs/ast/modules/Sect.tsx

```diff
@@ -0,0 +1,253 @@
+
+					const SECT_ABILITY = DIURNAL_SECT_STATUSES.includes(event.ability.guid) ? DIURNAL_SECT_BUFF_ABILITY : NOCTURNAL_SECT_BUFF_ABILITY
+
+					// Fab a sect buff event at the start of the fight
+					events.splice(0, 0, {
+						...event,
+						timestamp: startTime - 1,
+						type: 'applybuff',
+						ability: SECT_ABILITY,
+						targetID: event.sourceID,
+						targetIsFriendly: true,
+						})
+					break
+				}
+
+			} else if (event.timestamp - startTime >= GIVE_UP_THRESHOLD) {
+				// Just give up after GIVE_UP_THRESHOLD
+				break
+			} else {
+				continue
+			}
+
+		}
+
+		return events
+	}
+
+	private onCast(event: CastEvent) {
+		// If they casted a sect at anytime it means they pulled without one on
+		this.pullWithoutSect = true
+		const sect = getDataBy(ACTIONS, 'id', event.ability.guid)
+		if (sect) {
+			this.activeSectId = sect.id
+		}
+	}
+
+	// Looking for the sect buff that we fabricated, but if they switched it on mid-fight that'll get picked up too
+	private onApplySect(event: BuffEvent) {
+		if (!this.activeSectId) {
+			const sect = getDataBy(ACTIONS, 'id', this.mapBuffToCast(event.ability.guid))
+			if (sect) {
+				this.activeSectId = sect.id
+			}
+		}
+	}
+
+	private onComplete() {
+
+		/*
+			SUGGESTION: Pulled without Sect
+		*/
+		if (this.pullWithoutSect || !this.activeSectId) {
+			this.suggestions.add(new Suggestion({
+				icon: !this.activeSectId || ACTIONS.DIURNAL_SECT.id === this.activeSectId ? ACTIONS.DIURNAL_SECT.icon : ACTIONS.NOCTURNAL_SECT.icon,
+				content: <Trans id="ast.sect.suggestions.no-sect.content">
+					Don't start without <ActionLink {...ACTIONS.DIURNAL_SECT} /> or <ActionLink {...ACTIONS.NOCTURNAL_SECT} />. There are several abilities that can't be used without one of these stances.
+				</Trans>,
+				why: <Trans id="ast.sect.suggestions.no-sect.why">
+					There was no sect detected at the start of the fight.
+				</Trans>,
+				severity: SEVERITY.MAJOR,
+			}))
+		}
+
+		/*
+			SUGGESTION: Noct with Scholar
+		*/
+		const combatants = this.combatants.getEntities()
+		for (const [key, combatant] of Object.entries(combatants)) {
+
+			if (combatant.type === 'LimitBreak') {
+				continue
+			}
+			this.partyComp.push(combatant.type)
+		}
+
+		if (this.activeSectId === ACTIONS.NOCTURNAL_SECT.id && this.partyComp.includes('Scholar')) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.NOCTURNAL_SECT.icon,
+				content: <Trans id="ast.sect.suggestions.noct-with-sch.content">
+					It is counter-productive to use this Sect with this composition. The main shields <StatusLink {...STATUSES.NOCTURNAL_FIELD} /> from <ActionLink {...ACTIONS.NOCTURNAL_SECT}/> do not stack with Scholar's main shield <StatusLink {...STATUSES.GALVANIZE} />.
+				</Trans>,
+				why: <Trans id="ast.sect.suggestions.noct-with-sch.why">
+					Nocturnal Sect was used with a Scholar in the party.
+				</Trans>,
+				severity: SEVERITY.MAJOR,
+			}))
+		}
+
+		if (this.activeSectId) {
```

Comment:
Might be worth a third state for "No sect detected", WDYT? Means the statistic is always visible, showing what's being used by the system.

Reply:
I like this

Reply:
"That" icon